study.md

```diff
@@ -93,15 +118,17 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.map { |array| array ** 2  }
 ```
```

Comment:
`array.map { |array_element| array_element ** 2 }`
You are passing array into your block, but what's in that block should represent each individual element on the array.